### DISCUSSION TITLE
Catalyst Integrations are OPEN - Let's build the future of Catalyst together!

### DISCUSSION THREAD

**Discussion by katiehoesley** [bigcommerce/catalyst#1303] [Announcements]
## 🚀 Catalyst Integrations: Let's build the future of Catalyst together! 🚀

Integrations into Catalyst are OPEN and we’re on the hunt for more! Our existing list of completed integrations includes: 
* Algolia
* Vercel
* Makeswift 
* Contentful 
* Mega Menu Builder from Hypa Apps 
* Klaviyo
* <Your integration here!> 

If you’re interested in building an integration for Catalyst, the documentation you’ll need is [here](https://www.catalyst.dev/docs#integrations). 

## What we’ll do for new integrations: 
* Once approved, we’ll promote your integration via our developer social channels & add it to our list of available integrations on [catalyst.dev](https://www.catalyst.dev/)
* If you’d like your integration to be featured in the Resource Hub, we’ll work together on content to publish ASAP 
* Work together to coordinate educational resources/user documentation for developers who will use your integration in the future 

If you have questions or want to meet with someone on our team to discuss your plans, please comment on this Discussion or send an email to [composablecoe@bigcommerce.com](mailto:composablecoe@bigcommerce.com)!

#LFG 🚀

**Comment by Fefe-Hern** [comment]
Are you able to chain integrations into one command, or only one is allowed and the rest should be manually setup?

**Reply by becomevocal** [reply]
At the moment you'd start with one and add the rest by looking at the integrations/* branches for a reference. The manifest files are there to enable a path forward for more complex integration workflows though.

What kind of integrations are you thinking you'd like to run all at once? Is there a specific search / CMS combo, as an example?
